src/main/java/br/com/eventvs/api/resources/EventoResource.java

```diff
@@ -65,6 +65,21 @@
 		return ResponseEntity.ok(eventoResponse);
 	}
 	
+	/***
+	 * Cancela um evento
+	 * @param eventoId
+	 * @return ResponseEntity
+	 */
+	@PatchMapping(value = PATH_EVENTO_ID_CANCELAR)
+	@ResponseStatus(HttpStatus.OK)
+	public ResponseEntity cancelarEvento(@PathVariable Integer eventoId) {
+		String email = eventvsSecurity.getPessoaEmail();
+		boolean status = gerenciarEventoControle.cancelarEvento(eventoId, email);
+		if(status) {
+			return ResponseEntity.ok().build();
+		}
+		return ResponseEntity.badRequest().build();
```

Comment:
coloca so o retorno com o ok

Reply:
O retorno com o ok ta dentro do if, então o java da erro pq se não entrar no if não tem retorno

Reply:
eu sei po, to dizendo pra tirar esse bad request daí, ou vc n tratou os erros no controlador?